docs/tutorial.md

```diff
@@ -368,20 +368,30 @@ _Output_:
 
     replacement for gate CNOT does not preserve the quantum state
 
+##### _CNOT to CZ decomposer_
+
+The decomposition of the `CNOT` gate into a `CZ` gate (with additional single-qubit gates) is used frequently.
+To this end a `CNOT2CZDecomposer` has been implemented that decomposes any `CNOT`s in a circuit to a
+`Ry(-π/2)`-`CZ`-`Ry(π/2)`; the decomposition is illustrated in the image below.
+
+<p align="center"> <img width="600" src="_static/cnot2cz.png"> </p>
+
+`Ry` gates are used instead of, _e.g._, `H` gates,
+as they are generally already more likely to be supported by target backends.
```

Comment:
Suggestion: `as they are, generally, more likely to be supported already by target backends.`

`Generally already more likely` sounds a bit weird to me.

Reply:
Ah, I see  you changed this later. Maybe you still want to add the `already`.